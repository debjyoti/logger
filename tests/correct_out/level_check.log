 Test INFO message 1
 WARN - Test WARN message 2
 ERROR - Test ERROR message 3
