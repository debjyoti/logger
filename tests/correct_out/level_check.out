
 New logger thread # 66592 created 


 Log file '../out/level_check.log' opened


 WARN - Test WARN message 2


 ERROR - Test ERROR message 3


 Closed log file


 Logger thread terminated

